src/main/java/com/epam/DAO/PersonService.java

```diff
@@ -0,0 +1,33 @@
+package com.epam.DAO;
+
+import com.epam.model.Person;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+
+
+import java.sql.ResultSet;
+import java.util.List;
+
+@Service
+public class PersonService {
+
+
+    private final NamedParameterJdbcTemplate jdbcTemplate;
+
+    @Autowired
+    public PersonService(NamedParameterJdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public List<Person> getPersons(){
+        return jdbcTemplate.query("SELECT * FROM person",
+                (ResultSet rs, int rowNum) -> {
```

Comment:
We dont need to use ResultSet and int in lambdas - we can just name params.
Also, it is highly recommended to encapsulated such lambdas in methods and use method reference type of lambdas (it increases readability and testing)

Reply:
https://habr.com/ru/post/224593/

Reply:
https://www.youtube.com/watch?v=7Iy1hVEXxsU